packages/terra-form-radio/tests/jest/RadioField.test.jsx

```diff
@@ -1,70 +1,119 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
+import uniqueid from 'lodash.uniqueid';
 
 /* eslint-disable-next-line import/no-extraneous-dependencies */
 import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
 import RadioField from '../../src/RadioField';
 import Radio from '../../src/Radio';
 
+jest.mock('lodash.uniqueid');
```

Comment:
We cam define the mock once here globally:
```suggestion
jest.mock('lodash.uniqueid');
uniqueid.mockReturnValue('uuid123');
```

Reply:
we need to import uniqueid, otherwise it'll throw an error.